CRM/Report/Form/Mailing/Summary.php

```diff
@@ -301,7 +301,7 @@ public function mailing_select() {
     $mailing->query($query);
 
     while ($mailing->fetch()) {
-      $data[mysql_real_escape_string($mailing->name)] = $mailing->name;
+      $data[$mailing->escape($mailing->name)] = $mailing->name;
```

Comment:
IMO, it's better to use CRM_Core_DAO::escapeString instead of calling DB_DataObject->escape directly